### PR TITLE
Addresses issue #66

### PR DESCRIPTION
**Pull request by czucca**
added extract function for gnls objects

### CODE REVIEW

File: vignettes/texreg.Rnw [hunk 1]
```diff
@@ -1,1343 +1,1344 @@
...
-extract.lm <- function(model) {
-  s <- summary(model)
-  names <- rownames(s$coef)
-  co <- s$coef[, 1]
-  se <- s$coef[, 2]
-  pval <- s$coef[, 4]
-
-  rs <- s$r.squared
-  adj <- s$adj.r.squared
-  n <-
```
Comment (leifeld) [inline vignettes/texreg.Rnw]: Aren't they called `Generalized non-linear least squares` models? That's at least compatible with the GNLS abbreviation.

Comment (czucca) [reply vignettes/texreg.Rnw]: yep, I corrected it. Sorry for the inversion!

**Comment by czucca** [comment]
yes I double checked: the output is completely identical. I'll take care of the conflicts and then it should be good to go.

**Comment by leifeld** [comment]
Thanks. Let me know when you are done with the conflicts and the revision, and I'll approve it!

File: vignettes/texreg.Rnw [hunk 2]
```diff
@@ -478,6 +477,7 @@ Class			& Package			& Description \\
 `\code{glmmadmb}'		& \pkg{glmmADMB}		& Generalized linear mixed models and extensions \\
 `\code{glmrob}  		& \pkg{robustbase}		& Robust fitting of generalized linear models \\
 `\code{gls}'			& \pkg{nlme}			& Generalized least squares \\
+`\code{gnls}'			& \pkg{nlme}			& Generalized nonlinear least squares  \\
```
Comment (leifeld) [inline vignettes/texreg.Rnw]: `nonlinear` -> `non-linear`. Thanks!

**Pull request merged**